api/src/test/java/bio/terra/tanagra/aousynthetic/MeasurementEntityQueriesTest.java

```diff
@@ -143,4 +143,37 @@ void generateSqlForHierarchyPathOfAMeasurementEntity() throws IOException {
     GeneratedSqlUtils.checkMatchesOrOverwriteGoldenFile(
         generatedSql, "aousynthetic/hierarchy-path-of-a-measurement.sql");
   }
+
+  @Test
+  @DisplayName(
+      "correct SQL string for listing all measurement entity instances that are root nodes in the hierarchy")
+  void generateSqlForAllRootNodeMeasurementEntities() throws IOException {
+    // filter for "measurement" entity instances that have t_path_concept_id IS NULL
+    // i.e. measurement root nodes
+    ApiFilter isRootNode =
+        new ApiFilter()
+            .binaryFilter(
+                new ApiBinaryFilter()
+                    // not setting AttributeValue means to use a null value
```

Comment:
Existing issue, but looking at this, I'm wondering if a more explicit NULL signal in AttributeValue would be cleaner. The current encoding feels a bit hard to distinguish from a malformed request.

Reply:
Agreed. I'll plan to tackle that in a future PR.